compile.sh

```diff
@@ -93,23 +92,23 @@ if [ "$MISSING_PACKAGES" != "" ]; then
 
 	# apt-get guide.
 	apt-get --help > /dev/null 2> /dev/null && \
-	missingDepsExit "sudo apt-get install$MISSING_PACKAGES"
+	missingDepsExit "sudo apt-get install $MISSING_PACKAGES"
```

Comment:
These were stuck together for a reason. It's because the printf above prints `printf %s "<SPACE>$PACKAGE_NAME"`